### ISSUE TITLE
Add support of `$permissions` rules

### ISSUE THREAD

**Issue by slavaleleka** [AdguardTeam/FiltersCompiler#186]
such rules should not be ditched by the compiler:
```
Error: "Unknown modifier: permissions=autoplay=()" in the rule: "$permissions=autoplay=()"
$permissions=autoplay=()
```

**Comment by Alex-302** [comment]
It should be excluded for unsupported platforms.

**Comment by slavaleleka** [comment]
already fixed now

released in [v1.1.104](https://github.com/AdguardTeam/FiltersCompiler/blob/master/CHANGELOG.md#v11104---2023-11-08)
